admin.py

```diff
@@ -121,7 +121,9 @@ def _SaveData(self, data, date):
           date=date,
           #hits=num_hits,
           #total_pages=total_pages,
-          day_percentage=rate
+          day_percentage=bucket_dict['rate']
```

Comment:
is a `rate` property guaranteed?

Reply:
Yes, if a bucket exists in the results at all, it'll be guaranteed to have `rate`, `milestone` and `low_volume` properties.